src/snapred/backend/data/Indexer.py

```diff
@@ -277,24 +242,20 @@ def indexPath(self):
         """
         return self.rootDirectory / f"{self.indexerType}Index.json"
 
-    def recordPath(self, version: Optional[int] = None):
+    def recordPath(self, version: int):
```

Comment:
Should this type be `Version`?

Reply:
Disk should only have a flattened int representation, so int is probably best.